### ISSUE TITLE
[jaeger-operator] Jaeger Helm Chart - Using Jaeger CRDs

### ISSUE THREAD

**Issue by oliverbaehler** [jaegertracing/helm-charts#114]
We manage all our releases through helm. For us it would make sense to have a very simple Helm-Chart which allows to create Jeager CRDs which are managed by the jaeger operator. Similar to the Implementation for the jaeger-operator Helm-Chart (https://github.com/jaegertracing/helm-charts/blob/master/charts/jaeger-operator/templates/jaeger.yaml). We have a multi-tenancy-scenario and want to be able to deploy multiple Jaeger instances on different tenants on the same cluster, which are all managed by one Jaeger-Operator instance per cluster. Therefor i am asking if it would make sense to have a wrapper chart to deploy a Jaeger Instance without the operator. Or are there other approaches used to fullfill such a multi-tenancy-scenario?

**Comment by naseemkullah** [comment]
You want to deploy your Jaeger-Operator managed Jaegers(CRDs) through a helm chart, is that it?

I do not use the jaeger operator currently so am unqualified to answer.

@cpanato @batazor thoughts?

**Comment by oliverbaehler** [comment]
@naseemkullah correct, just a chart that implements Jeager CRDs. I think the possibility to create one Jeager Instance per Chart would be enough. I could make a contribution for that. Let's w8 for some thoughts =)

**Comment by bsundsrud** [comment]
We manage the CRDs using the [`incubator/raw`](https://github.com/helm/charts/tree/master/incubator/raw) chart and hand it a values file that looks something like this:
```
resources:
  - apiVersion: jaegertracing.io/v1
    kind: Jaeger
    metadata:
      name: jaeger-instance
    spec:
      ...
```
